internal/tiltfile/config/bool.go

```diff
@@ -0,0 +1,72 @@
+package config
+
+import (
+	"bytes"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"strconv"
+
+	"go.starlark.net/starlark"
+)
+
+type boolSetting struct {
+	value bool
+	isSet bool
+}
+
+var _ configValue = &boolSetting{}
+var _ flag.Value = &boolSetting{}
+
+func (s *boolSetting) starlark() starlark.Value {
+	return starlark.Bool(s.value)
+}
+
+func (s *boolSetting) IsSet() bool {
+	return s.isSet
+}
+
+func (s *boolSetting) MarshalJSON() ([]byte, error) {
+	ret := &bytes.Buffer{}
```

Comment:
Is this equivalent to json.Marshal(s.value)?

Reply:
yes, AND we don't need to marshal anyway - this is copypasta of cruft left over from when we were persisting these values